StudCity.Frontend/src/pages/registration/Registration.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import Label from "../../UI/Label";
+import TextField from "../../UI/fields/TextField";
+import PasswordTextField from "../../UI/fields/PasswordTextField";
+import Button from "../../UI/Button";
+import GoogleButton from "../../UI/GoogleButton";
+import { passwordMatchValidation, passwordValidation } from "../../utils/validators/validators";
+
+function Registration() {
+	const [disabled, setDisabled] = useState(false)
+	const [passwords, setPasswords] = useState({
+		password: "",
+		confirmPassword: ""
+	})
+	
+	return (
+		<div className="w-1/1 h-screen flex">
+			<div className="w-1/2 h-screen ">
+				<img className="w-full h-screen object-cover" src="/images/Image-Registration-Light.jpg" alt="404" />
+			</div>
+			<div className="form w-1/2 h-screen flex bg-primatyWhite">
+				<div className="container flex flex-col w-4/5 m-auto h-[95%] bg-primatyWhite rounded-3xl p-11 shadow-md ">
+					<h1 className="w-full text-center font-bold text-4xl my-3 text-primaryRegistration">Welcome</h1>
+					<h6 className="w-full text-center font-normal text-lg text-primaryRegistration">Please enter your details</h6>
+					<form className="flex flex-col w-1/1">
+						<Label>Email</Label>
+						<TextField 
+							placeholder = "Enter your email" 
+							className = "" 
+							type = "email" 
+							required = {true}
+						/>
+						<Label>Password</Label>
+						<PasswordTextField 
+							placeholder = "Enter your password"
+							className = "" 
+							required = {true}
+							setDisabled = {setDisabled}
+							handleChange = {(evnt) => setPasswords({ ...passwords, 
```

Comment:
please create a function at the top of this component, and pass it as a prop
PS: in other places as well